WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -1002,6 +1037,8 @@ public string AmountWatermarkText
 			set => this.RaiseAndSetIfChanged(ref _amountWaterMarkText, value);
 		}
 
+		public bool IsCustomFee => Global.UiConfig.CustomFee is true;
```

Comment:
It is not refreshed immediately if I change it in SettingsTab.

Reply:
Unsure why this is happening, if someone could help with this I'd really appreciate it

Reply:
Did.